pkg/grpc/grpc.go

```diff
@@ -58,25 +60,43 @@ func NewGRPCClientFromConfiguration(configuration *configuration.ClientConfigura
 		dialOptions = append(dialOptions, grpc.WithInsecure())
 	}
 
+	if oauthConfig := config.Oauth; oauthConfig != nil {
+		var perRPC credentials.PerRPCCredentials
+		var err error
+		switch oauthConfig.Credentials.(type) {
+		case (*configuration.ClientOAuthConfiguration_GoogleDefaultCredentials):
```

Comment:
Nit: You can remove the parentheses here.

Reply:
sure :)